src/main/java/Exercise10.java

```diff
@@ -0,0 +1,11 @@
+public class Exercise10 {
+    public String getDay(int day, int month, int year) {
+        int JMD = (day + ((153 * (month + 12 * ((14 - month) / 12) - 3) + 2) / 5) +
```

Comment:
Tên biến không rõ ràng, không theo convention

```suggestion
        int JMD = (day + ((153 * (month + 12 * ((14 - month) / 12) - 3) + 2) / 5) +
```